packages/application-shell/src/components/user-settings-menu/user-settings-menu.tsx

```diff
@@ -151,29 +169,30 @@ const UserSettingsMenuBody = props => {
             </div>
           </Spacings.Inline>
         </Spacings.Inset>
-        {menuLinks.map(menu => (
-          <OptionalFeatureToggle
-            key={menu.key}
-            featureToggle={menu.featureToggle}
-          >
-            <Link
-              to={`/account/${menu.uriPath}`}
-              onClick={props.downshiftProps.toggleMenu}
+        {applicationsAppBarMenu &&
+          applicationsAppBarMenu.map(menu => (
+            <OptionalFeatureToggle
+              key={menu.key}
+              featureToggle={menu.featureToggle}
             >
-              <MenuItem>
-                <Spacings.Inset scale="s">
-                  {renderLabel(menu, props.locale)}
-                </Spacings.Inset>
-              </MenuItem>
-            </Link>
-          </OptionalFeatureToggle>
-        ))}
+              <Link
+                to={`/account/${menu.uriPath}`}
+                onClick={() => props.downshiftProps.toggleMenu()}
+              >
+                <MenuItem>
+                  <Spacings.Inset scale="s">
+                    {renderLabel(menu, props.language)}
+                  </Spacings.Inset>
+                </MenuItem>
+              </Link>
+            </OptionalFeatureToggle>
+          ))}
         <MenuItem hasDivider={true} />
         <a
           href={`https://commercetools.com/privacy#suppliers`}
           target="_blank"
           rel="noopener noreferrer"
-          onClick={props.downshiftProps.toggleMenu}
+          onClick={() => props.downshiftProps.toggleMenu()}
```

Comment:
what's the difference in here? 🤔

Reply:
Simply that the HTML `event` is not "compatible" with the expected argument of `toggleMenu` 🤷‍♂️